app/gen-server/migration/1663851423064-UserUUID.ts

```diff
@@ -15,12 +14,15 @@ export class UserUUID1663851423064 implements MigrationInterface {
 
     // Updating so many rows in a multiple queries is not ideal. We will send updates in chunks.
     // 300 seems to be a good number, for 24k rows we have 80 queries.
-    const userList = await queryRunner.manager.createQueryBuilder()
-      .select("users")
-      .from(User, "users")
-      .getMany();
-    userList.forEach(u => u.ref = makeId());
-    await queryRunner.manager.save(userList, { chunk: 300 });
+    const userList = await queryRunner.query("SELECT * FROM users;");
+    let transaction = "";
+    for (let i = 0; i < userList.length; i += 1) {
```

Comment:
Maybe we could group the users into sub-arrays of size 300 instead? Then, each group of 300 could run serially in a single transaction, with the 300 updates done in parallel. Something like:

```javascript
const userChunks = chunk(userList, 300);
for (const users of userChunks) {
  await queryRunner.connection.transaction(async manager => {
    const queries = users.map(user => {
      return manager.query(`UPDATE users SET ref = '${makeId()}' WHERE id = ${user.id}`);
    });
    await Promise.all(updateTasks);
  });
}
```

Here's how TypeORM chunks entities - doesn't have to be done exactly this way, but it's a useful reference: https://github.com/typeorm/typeorm/blob/e7649d2746f907ff36b1efb600402dedd5f5a499/src/util/OrmUtils.ts#L11

Parameterizing the queries would be safer, but I think there's some nuance to how you do it with sqlite vs. Postgres. I don't see any obvious SQL injection risks, but I haven't looked very hard. If it's not hard to do, I think we should do it.

@berhalak could you also do a review of the changes to the migrations? Thanks.